cmd/clusterctl/test/README.md

```diff
@@ -1,26 +1,28 @@
-# Running the tests
+# Clusterctl E2E tests
 
-NOTE: the e2e tests may not work on a Mac; they should however work just fine on any Linux distro. Mac support will be added in a follow-up PR.
-
-Currently, overrides are needed for the cluster-api, kubeadm-bootstrap and kubeadm-control-plane providers. The override for the infra provider docker must be removed as it is generated locally by the e2e test script:
-
-	cmd/clusterctl/hack/local-overrides.py
-	rm -rf $HOME/.cluster-api/overrides/docker
+The e2e test framework is driven by a config file, so you can easily switch images/manifests from dev/ci or prod,
+ switch the infrastructure provider, use different workload cluster templates.
 
 The entire test suite can be run using the script:
 	
-	./run-e2e.sh
+	cmd/clusterctl/test/run-e2e.sh
+
+To run specific suite/spec, use the `GINKGO_FOCUS` os environment variable:
+
+	GINKGO_FOCUS="init and create workload cluster (suite-1)" cmd/clusterctl/test/run-e2e.sh
+
+To select a specific test configuration, use the `E2E_CONF_FILE` os environment variable:
 
-To run specific tests, use the `GINKGO_FOCUS`
+	E2E_CONF_FILE="~/my-e2e.conf" cmd/clusterctl/test/run-e2e.sh
 
-	GINKGO_FOCUS="clusterctl create cluster" ./run-e2e.sh
+To specify where test artifacts should be stored, use the `ARTIFACTS` os environment variable:
 
-## Skip local build of CAPD
+	ARTIFACTS="~/_artifacts" cmd/clusterctl/test/run-e2e.sh
```

Comment:
Can we add this directory to the .gitignore file list?

Reply:
I guess you mean _artifacts under the cluster-api root
done